tensorflow/1.10/README.md

```diff
@@ -58,7 +58,7 @@ configuration access.
 <a name="service-name"></a>
 ### Service Name
 
-Each instance of TensorFlow in a given DC/OS cluster must be configured with a different service name. You can configure the service name in the **service** section of the advanced installation section of the DC/OS web interface. The default service name (used in many examples here) is _`tensorflow`_.
+Each instance of TensorFlow in a given DC/OS cluster must be configured with a different service name. You can configure the service name in the **service** section of the advanced installation section of the DC/OS web interface. The default service name (used in many examples here) is _`beta-tensorflow`_.
 
 <a name="job-url"></a>
 ### Job URL
```

Comment:
I can't comment on line 91 since it wasn't edited in this PR, but on that line we reference "Using Google Cloud Storage" which doesn't exist on this page. We should probably link that to https://github.com/dcos-labs/dcos-tensorflow-tools#using-google-cloud-storage-optional.

Reply:
Added